compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.h

```diff
@@ -21,6 +21,15 @@ setDataTiledMultiMmaLoweringConfig(IREE::GPU::TargetAttr target,
                                    mlir::FunctionOpInterface entryPoint,
                                    Operation *op);
 
+/// Helper for setting up a convolution config using IGEMM based on the
+/// specified target.
+/// TODO: Currently this only succeeds if the target supports an mma
```

Comment:
We might not need this TODO actually. IGEMM might not be worth it without a special mma instruction.

Reply:
I'll leave it for now and then we can reevaluate later whether we should do this.